examples/SelectInForm/Main.elm

```diff
@@ -0,0 +1,115 @@
+module SelectInForm.Main exposing (main)
```

Comment:
You could just add a form to the exiting example so there is not more code that we need to maintain. Or is there something that will not be tested or is different here?